Examples/Python/python/acts/examples/dd4hep.py

```diff
@@ -0,0 +1,27 @@
+import multiprocessing
+
+
+# Cannot conveniently catch linker errors, so we launch a suprocess to
+# try importing and see if it works in order to provide a useful error message
+def _import_test():
+    from acts import ActsPythonBindingsDD4hep
+
+
+p = multiprocessing.Process(target=_import_test)
+p.start()
+p.join()
+if p.exitcode != 0:
+    raise RuntimeError(
+        "Error encountered importing DD4hep. Likely you need to set LD_LIBRARY_PATH."
+    )
+
```

Comment:
Those lines caused a crash when I tried to run the geometry script. They should probably be removed.

Reply:
This is supposed to provide some help when you try to use dd4hep but don't have it set up. In that case, it will simply die, without any error at all, and you won't know what the cause is. I'll try out another approach.